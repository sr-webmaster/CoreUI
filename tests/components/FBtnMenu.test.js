import { mount } from '@vue/test-utils'
import { FIXTURE_UI_BTNMENU_ITEMS } from '@freshinup/core-ui/tests/__data__/BtnMenu/items'
import Component from '@freshinup/core-ui/src/components/FBtnMenu.vue'

const items = FIXTURE_UI_BTNMENU_ITEMS

describe('Button Menu', () => {
  it('snapshot default', async () => {
    const wrapper = mount(Component)
    expect(wrapper.element).toMatchSnapshot()
  })
  it('snapshot with items', async () => {
    const wrapper = mount(Component, {
      propsData: {
        items
      }
    })
    expect(wrapper.element).toMatchSnapshot()
  })
})
