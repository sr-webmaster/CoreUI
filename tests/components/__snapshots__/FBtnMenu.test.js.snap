// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button Menu snapshot default 1`] = `
<div
  class="v-menu"
>
  <button
    class="v-btn v-btn--disabled theme--light primary--text"
    disabled="disabled"
    type="button"
  >
    <div
      class="v-btn__content"
    >
      Menu 
      <!---->
    </div>
  </button>
  <div
    class="v-menu__content theme--light "
    style="max-height: auto; min-width: 0px; max-width: auto; top: 12px; left: 0px; transform-origin: top left; z-index: 0; display: none;"
  >
     
    <div
      class="v-list theme--light"
      role="list"
    />
  </div>
</div>
`;

exports[`Button Menu snapshot with items 1`] = `
<div
  class="v-menu"
>
  <button
    class="v-btn theme--dark primary primary-text--text"
    type="button"
  >
    <div
      class="v-btn__content"
    >
      Menu 
      <i
        aria-hidden="true"
        class="v-icon v-icon--right material-icons theme--dark"
      >
        arrow_drop_down
      </i>
    </div>
  </button>
  <div
    class="v-menu__content theme--light "
    style="max-height: auto; min-width: 0px; max-width: auto; top: 12px; left: 0px; transform-origin: top left; z-index: 0; display: none;"
  >
     
    <div
      class="v-list theme--light"
      role="list"
    >
      <div
        role="listitem"
      >
        <a
          class="v-list__tile v-list__tile--link theme--light"
        >
          <div
            class="v-list__tile__title"
          >
            Item 1
          </div>
        </a>
      </div>
      <div
        role="listitem"
      >
        <a
          class="v-list__tile v-list__tile--link theme--light"
        >
          <div
            class="v-list__tile__title"
          >
            Item 2
          </div>
        </a>
      </div>
    </div>
  </div>
</div>
`;
